src/drone/cli.rs

```diff
@@ -46,6 +46,18 @@ pub struct Opts {
     #[clap(long, action)]
     pub acme_server: Option<String>,
 
+    //email to use as mailto for cert issuance
```

Comment:
Use a proper docstring so that this appears in the `--help` message of the CLI:

```
/// Email to use as mailto: for ACME certificate issuance.
```

Note the three `///`; without this, `clap` won't pick this up as cli documentation.